### ISSUE TITLE
`balena push` randomly stops streaming logs when pushing in local mode

### ISSUE THREAD

**Issue by Ereski** [balena-io/balena-cli#2344]
Still needs more investigation and seems to be random. Also only seems to happen with JF.

cc @Hades32

**Comment by Ereski** [comment]
Not sure if related but I see multiple errors like this:

```
Sep 22 12:20:34 7d60c99 agetty[3237]: /dev/ttyS0: not a tty
```

**Comment by srlowe** [comment]
@Ereski It's possible that this was fixed by Lucian's recent changes - would you be able to test with latest?

**Comment by Ereski** [comment]
@srlowe it's pretty random. If I don't see it again it might have been solved, maybe?

**Comment by srlowe** [comment]
@Ereski Yeah it's quite possible it was fixed.  Ok, let's keep the issue open for a while.  Please let me know if it happens again.
